### DISCUSSION TITLE
Feature Request: Changeable keybinds

### DISCUSSION THREAD

**Discussion by WeeboChan** [cafali/SnapKey#22] [New Features & Ideas]
I have a problem where I have an EU keyboard, and I was wondering if I could somehow change keybinds?

Instead of the US version where they use A/D I would like to change keybinds to Q/D as example.

**Comment by WeeboChan** [comment]
Ah I see, seems like there already was a request.

My apologies.

**Comment by cafali** [comment]
Key binding is now available!

1.1.3 - 07/2024 (latest)

Key Bindings:
Introducing customizable key bindings to accommodate various keyboard layouts such as AZERTY (where A=Q, D=D). You can now modify the keys managed by SnapKey by editing the config.cfg file using the ASCII Code List.

Others:
Updated Readme.pdf
